python-dockerfile-parse.spec

```diff
@@ -121,7 +121,7 @@ py.test-%{python3_version} -v tests
 * Tue Jun 02 2020 Robert Cerven <rcerven@redhat.com> 0.0.18-1
 - new upstream release: 0.0.18
 
-* Fri Apr 24 2020 Martin Bašti <mbasti@redhat.com> 0.0.17-1
+* Fri Apr 24 2020 Martin Basti <mbasti@redhat.com> 0.0.17-1
```

Comment:
Pls add this change to separate commit

Reply:
By "separate commit", do you mean a different Pull Request? Or a separate commit within this Pull Request?